content/kubermatic/master/startio/cheat_sheets/access_to_cluster/_index.en.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Get Access to Kubernetes Cluster"
```

Comment:
This guide should have a prerequisite section specifying that you need Terraform and KubeOne locally with links to relevant installation guides. I know that we already specify tools used in the pipeline in the concepts documents, but it might be useful to specify which tools you need locally and how to get them for each guide.